tests/filecheck/parser-printer/parse_error.mlir

```diff
@@ -14,3 +14,11 @@ test.op : () -> ()
 // CHECK-NEXT: test.op : () -> ()
 // CHECK-NEXT:         ^
 // CHECK-NEXT:         Operation test.op does not have a custom format.
+
+// -----
+
+module {
+    "test.op"() [^unknown_successor]: () -> ()
+}
+
+// CHECK: Unknown location of span region ends with missing block declarations for block(s) unknown_successor.
```

Comment:
"span" is parser terminology, no? Same with "unkown location".

The block name could be in quotes - maybe have a look how we do it in other block references within error messages.

Reply:
This error message is definitely not correct.

Reply:
why is it a problem to use parser terminology here (since it's a parse error)?

Reply:
how would I know what the correct error might be? (this one is the one that's currently being printed when the code is run)

Reply:
I'm surprised that this is the error printed as the location should now be known.